.vscode/.ropeproject/config.py

```diff
@@ -0,0 +1,114 @@
+# The default ``config.py``
```

Comment:
Create .gitignore file in your project to ignore some of these settings specific to your development environment.
Directories like `.vscode`, `__pycache__`, `.idea` should be ignored.